lib/cmd/fh3/appdata/import/start.js

```diff
@@ -0,0 +1,87 @@
+"use strict";
+/* globals i18n */
+
+var async = require('async');
+var path = require('path');
+var mime = require('mime');
+
+var fs = require('fs')
+  , fhreq = require('../../../../utils/request.js')
+  , readline = require('readline')
+  , common = require('../../../../common');
+
+function retrieveUploadMetadata(envId, appId, filePath, cb) {
+  var url = 'api/v2/appdata/' + envId + '/' + appId + '/data/import';
+
+  fs.stat(filePath, function (err, stat) {
+    if (err) {
+      return cb(err);
+    }
+
+    var fields = {
+      filename: path.basename(filePath),
+      filesize: stat["size"]
+    };
+    // Get the jobID
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, fields, i18n._('Error importing data: '), function (err, jobData) {
+      if (err) {
+        return cb(err);
+      }
+
+      return cb(null, jobData);
+    });
+  });
+}
+
+function sendFile(filePath, jobData, cb) {
+  var uploadUrl = jobData.url;
+  var filename = path.basename(filePath);
+  var mimetype =  mime.lookup(filename);
+  fhreq.uploadFile(uploadUrl, filePath, {}, mimetype, function(err) {
+    if (err) {
+      return cb(err);
+    }
+
+    cb(null, jobData.jobId);
+  });
+}
+
+function startJob(envId, appId, filePath, cb) {
+  async.waterfall([
+    async.apply(retrieveUploadMetadata, envId, appId, filePath),
+    async.apply(sendFile, filePath)
+  ], function(err, jobId) {
+    cb(err, jobId);
+  });
+}
+
+module.exports = {
+  'desc': i18n._('Start a new data import job for the given application in the given environment'),
+  'examples': [{
+    cmd: 'fhc addpata import start --appId=5ujx2eifvzaudq43nw4nmvcu --envId=live --path=/my/import.tar',
+    desc: 'Start a new data import job for the given application in the given environment'
+  }],
+  'demand': ['envId', 'appId', 'path'],
+  'alias': {},
+  'describe': {
+    'envId': i18n._('Environment id'),
+    'appId': i18n._('Application id'),
+    'path': i18n._('path to the file to be uploaded')
+  },
+  'customCmd' : function (params, cb) {
+    startJob(params.envId, params.appId, params.path, function (err, jobId) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)